### PR TITLE
Use minutes instead of seconds where possible.

### PR DESCRIPTION
**Pull request by php-coder**
IMHO it's a bit easier to understand when you see `time.Minute` instead of `60*time.Second`. So, I did such replacements in tests. Interesting to hear your opinions, maybe it's not worth it.

PTAL @rhcarvalho @mnagy @bparees

### CODE REVIEW

File: test/extended/builds/start_build.go [hunk 1]
```diff
@@ -160,7 +160,7 @@ var _ = g.Describe("builds: parallel: oc start-build", func() {
 
 			g.By("getting the build name")
 			var buildName string
-			wait.Poll(time.Duration(100*time.Millisecond), time.Duration(60*time.Second), func() (bool, error) {
+			wait.Poll(time.Duration(100*time.Millisecond), time.Duration(time.Minute), func() (bool, error) {
```
Comment (rhcarvalho) [inline test/extended/builds/start_build.go]: No need for the type conversion? s/`time.Duration(time.Minute)`/`time.Minute` :smile:

Comment (php-coder) [inline test/extended/builds/start_build.go]: @rhcarvalho I've chaned only one what you've pointed and leave the rest (like `time.Duration(100*time.Millisecond`).

**Comment by rhcarvalho** [comment]
LGTM

**Comment by bparees** [comment]
in the future this is not something i'd recommend you spend time on, but it's fine i guess, since you're done.

one downside to doing this is is we decide we need to bump the timeout by 30s (for example) later, we have to revert this change, rather than just add 30s to the value.

[merge]

**Comment by php-coder** [comment]
@bparees Ok, thanks!

**Pull request merged**